tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -39,6 +39,26 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.UploadBatchSize"/>
             public const string UploadBatchSize = "DD_DEBUGGER_UPLOAD_BATCH_SIZE";
 
+            /// <summary>
+            /// Configuration key for the maximum symbol size to upload (in bytes).
+            /// Default value is 1 mb.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.SymbolBatchSizeInBytes"/>
+            public const string SymbolBatchSizeInBytes = "DD_DEBUGGER_SYMBOL_BATCH_SIZE_BYTES";
+
+            /// <summary>
+            /// Configuration key for allowing uploading symbol database.
+            /// Default value is false (disabled).
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.SymbolDatabaseUploadEnabled"/>
+            public const string SymbolDatabaseUploadEnabled = "DD_SYMBOL_DATABASED_UPLOAD_ENABLED";
+
+            /// <summary>
+            /// Configuration key for allowing uploading symbol database for internal use.
```

Comment:
```suggestion
            /// Configuration key for forcing the client library to upload symbol data, regardless of whether it is requested by our backend. For internal use and testing.
```

Reply:
Fixed in db6086b623f7ad8ae43181865be84140bce1ef3c